### ISSUE TITLE
Hi，How could I get the BLEU score，for example, specific command？

### ISSUE THREAD

**Issue by Markkk111** [Yuanhy1997/SeqDiffuSeq#18]
I should run “python bleu_eval.py   ... ...”

**Comment by 1754699523** [comment]
Are you slove it ?

**Comment by Yuanhy1997** [comment]
Of course, you can run 'python bleu_eval.py' and attach the path to your result json file. And if you want to use it to evaluate a series of result files, you can attach the result file directory path with the prefix of the files and you can also specify some specific name of the files in a following argument.

**Comment by 1754699523** [comment]
Can bleu_eval.py be used for non-translation tasks?

**Comment by Yuanhy1997** [comment]
Of course, BLEU is not a translation-specific metric, while for a fair comparison, I recommend you use the evaluation script in DiffuSeq for non-translation tasks since different packages for computing BLEU may result in very different scores.
